RELEASE.md

```diff
@@ -1,9 +1,95 @@
 # Release Process
 
-The Kubernetes Template Project is released on an as-needed basis. The process is as follows:
+## Overview
 
-1. An issue is proposing a new release with a changelog since the last release
-1. All [OWNERS](OWNERS) must LGTM this release
-1. An OWNER runs `git tag -s $VERSION` and inserts the changelog and pushes the tag with `git push $VERSION`
-1. The release issue is closed
-1. An announcement email is sent to `kubernetes-dev@googlegroups.com` with the subject `[ANNOUNCE] kubernetes-template-project $VERSION is released`
+The Network Policy API project is has the following two main release components:
+- Kubernetes Custom Resource Definitions (CRDs)
+- Corresponding Go API in the form of `sigs.k8s.io/network-policy-api` Go package
+
+This repository is the home for both of the above components.
+
+## Versioning strategy
+The versioning strategy for this project is covered in detail in [the release
+documentation].
+
+[the release documentation]: site-src/versioning.md
+
+## Releasing a new version
+
+### Writing a Changelog
+
+To simplify release notes generation, we recommend using the [Kubernetes release
+notes generator](https://github.com/kubernetes/release/blob/master/cmd/release-notes):
+
+```
+go install k8s.io/release/cmd/release-notes@latest
+export GITHUB_TOKEN=your_token_here
+release-notes --start-sha EXAMPLE_COMMIT --end-sha EXAMPLE_COMMIT --branch main --repo network-policy-api --org kubernetes-sigs
+```
+
+This output will likely need to be reorganized and cleaned up a bit, but it
+provides a good starting point. Once you're satisfied with the changelog, create
+a PR. This must go through the regular PR review process and get merged into the
+`main` branch. Approval of the PR indicates community consensus for a new
+release.
+
+### Release Steps
+
+The following steps must be done by one of the [network-policy API maintainers][network-policy-api-team]:
+
+For a **PATCH** release:
```

Comment:
> Is 0.1.0 a PATCH release or a MAJOR.MINOR release?

That would be a Minor release whole 0.1.1 would be a patch release.  I think we should include docs for both Patch and Major/Minor relases since If we do go for a v1apha2 API release I think it would be a "patch" release (Maybe I'm not 100% sure) 

> Also wondering if we should call out the reference to gateway-api here for giving credit/reference?

Added

Reply:
https://github.com/kubernetes/org/pull/4217

Reply:
> > Is 0.1.0 a PATCH release or a MAJOR.MINOR release?
> 
> That would be a Minor release whole 0.1.1 would be a patch release. I think we should include docs for both Patch and Major/Minor relases since If we do go for a v1apha2 API release I think it would be a "patch" release (Maybe I'm not 100% sure)

yeah me neither which is why I was like maybe we add at that point when we want to cross that bridge.. At this point we are cutting 0.1.0 and that is patch so let's keep that only? Because I thought v1alpha2 would just be 0.1.1 or something... Since gateway-api is complex they have so many types of this, we should rethink this for our needs since we will do a per API thingy, I doubt we will use major.minor release? Let's keep the process as simple as possible, but I won't hold this PR because you have extra docs for stuff we aren't using yet.

> 
> > Also wondering if we should call out the reference to gateway-api here for giving credit/reference?
> 
> Added

added where? :D I am going blind...

Reply:
ah found it! https://github.com/kubernetes-sigs/network-policy-api/pull/91/commits/25c0f29a3a15230e64d13378811b625072ba53d0#diff-4e63587151032fdaf6fb3011e6594e8a3b294309a92d67d15654cc04b98d1757R1